### ISSUE TITLE
Incomplete README

### ISSUE THREAD

**Issue by adityaraute** [sorenrehkopf/custom-rick-roll#22]
The Readme file does not very well explain how one can run this project. If a procedure was listed with steps to follow, it would really benefit the users.

**Comment by sorenrehkopf** [comment]
Hey @adityaraute!

I have not been aware that other people are trying to run this project at all actually. I linked the repo from the generated pages as a random decision years and years ago when I first set this up as a hobby project, and I've kept it there since people have been using it to report issues.

Tbh I'm not entirely sure why people would want to be running this though? This is just the client that people see when they go to secretrickroll.com and doesn't actually contain any of the page generation logic. 

I guess you're welcome to run it if you'd like though! I built this project a while ago so it uses Gulp for the build and dev server, and it may be incompatible with more recent versions of node actually. The latest node version while I was building it was v10 and that is what I had to switch to last time I ran it locally myself iirc.

In either case if you or anyone else viewing this issue is trying to run it locally then the rough steps you'll probably need are
1. Set node verision to 10.0.0.
2. Yarn install in the repo.
3. Install the gulp cli locally. I have v3.9.1 installed personlly and there is a high probability other version will have issues.
4. Run `gulp start` in the repo. 

And I think that should run it for you?

I'm gonna go ahead and close this issue out without updating the readme though as I haven't confirmed that those steps will work, and like I mentioned above, I'm confused as to why people would want to be running this themselves in the first place. If you're able to get it running on your machine and want to document your setup steps for others than you're more than welcome to PR an updated readme with your steps though! 

Sorry to not be more help, best of luck!
